src/components/AssetInfo/AssetInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { hexToNumberString } from "web3-utils";
+
+import { ETHERSCAN_BASE_URL } from "../../config";
+
+const makeEtherscanTokenURL = ({ registryAddress, tokenId }: ERC721TokenParameters) => {
+  const tokenIdDecimal = hexToNumberString(tokenId);
+  return `${ETHERSCAN_BASE_URL}token/${registryAddress}?a=${tokenIdDecimal}`;
+};
+
+interface ERC721TokenParameters {
```

Comment:
Suggestion:
- using Props instead of Parameters (it's component props)

Reply:
would prefer to keep parameters because it's not just for react